add stickers and another wallpaper

honchokomodo opened the pull request.

honchokomodo committed:
add assets

honchokomodo committed:
update attribution

honchokomodo committed:
Create .gitignore

honchokomodo committed:
rename files

honchokomodo committed:
update hyperlinks

honchokomodo committed:
add thinking

honchokomodo committed:
Merge branch 'main' of https://github.com/honchokomodo/juniper

honchokomodo committed:
add shook

honchokomodo committed:
update attribution

honchokomodo committed:
create stickers

honchokomodo committed:
oops spelling error

Sntx626 reviewed (approved):
Looks awesome,
beautiful work!

honchokomodo committed:
Merge branch 'main' of https://github.com/honchokomodo/juniper

honchokomodo committed:
add winter solstice art

created for the winter solstice. a bit late but ehhhhhhhhh
will include source kra file when i get around to lengthening her jacket.

honchokomodo commented:
will get around to adding source kra file eventually tm
